src/transformStateWithClones.js

```diff
@@ -8,6 +9,29 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const table = {
```

Comment:
[NAMING] - The name `table` for the object copy is not descriptive of its purpose. A better name would be `stateCopy` to clearly indicate that it is a copy of the `state` object.